build/yamls/windows/containerd-with-ovs/conf/Install-OVSDriver-Containerd.ps1

```diff
@@ -0,0 +1,18 @@
+$ErrorActionPreference = "Stop"
+mkdir -force c:/openvswitch/driver
+$mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+$mountPath = ($mountPath.Replace('\', '/')).TrimEnd('/')
+$OVSDriverDir = "$mountPath\openvswitch\driver"
+
+# Check if OVSExt driver is already installed
+$driverStatus = netcfg -q ovsext
+if ($driverStatus -like '*not installed*') {
```

Comment:
Do we have a plan to install certificates including the self-signed certs in the init-container when installing OVS driver?

Reply:
I recall our discussion on related topic: for the officially signed OVS, Antrea requires no additional actions. However, for other OVS releases, it is the user's responsibility to add the certificate before running Antrea. We have already highlighted this in the current documentation(https://github.com/gran-vmv/antrea/blob/ipam-e2e-k8s/docs/windows.md#1-optional-install-ovs-provided-by-antrea-or-your-own)